### DISCUSSION TITLE
How do I move Tab Groups?

### DISCUSSION THREAD

**Discussion by SeanPedersen** [lbragile/TabMerger#11] [Q&A] [answered]
Hey, I want to reorder my tab groups but drag n drop is not working? Is that a bug or a missing feature?

**Comment by SeanPedersen** [comment, marked as answer]
Ok just found the answer, it's on TODO list: https://github.com/lbragile/TabMerger/projects/1#card-52075725

**Reply by lbragile** [reply]
Great! Yes it is a todo list item 😀

**Reply by lbragile** [reply]
@SeanPedersen This functionality has been added since [v1.6](https://github.com/lbragile/TabMerger/releases/tag/v1.6.2)
